gradle/libs.versions.toml

```diff
@@ -1,28 +1,28 @@
 [versions]
-keycloakVersion = "26.0.7"
 amazonSoftwareVersion = "2.29.29"
-junitJupiterVersion = "5.11.3"
 apacheLoggingVersion = "2.24.2"
 bouncyCastleVersion = "1.79"
+junitJupiterVersion = "5.11.3"
+keycloakVersion = "26.0.7"
 
 [libraries]
 comAmazonaws-awsLambdaJavaCore = "com.amazonaws:aws-lambda-java-core:1.2.3"
 comAmazonaws-awsLambdaJavaEvents = "com.amazonaws:aws-lambda-java-events:3.14.0"
+comAmazonaws-awsLambdaJavaLog4j2 = "com.amazonaws:aws-lambda-java-log4j2:1.6.0"
 softwareAmazonAwssdk-cognitoidentityprovider = { group = "software.amazon.awssdk", name = "cognitoidentityprovider", version.ref = "amazonSoftwareVersion" }
-comFasterxmlJacksonDataformat-jacksonDataformatYaml = "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.18.2"
 softwareAmazonAwssdk-ssm = { group = "software.amazon.awssdk", name = "ssm", version.ref = "amazonSoftwareVersion" }
-orgKecloak-keycloakSamlCore = { group = "org.keycloak", name = "keycloak-saml-core", version.ref = "keycloakVersion" }
-orgKecloak-keycloakServices = { group = "org.keycloak", name = "keycloak-services", version.ref = "keycloakVersion" }
-orgBouncycastle-bcprovJdk18on = { group = "org.bouncycastle", name = "bcprov-jdk18on", version.ref = "bouncyCastleVersion" }
-orgBouncycastle-bcpkixJdk18on = { group = "org.bouncycastle", name = "bcpkix-jdk18on", version.ref = "bouncyCastleVersion" }
-orgApacheLoggingLog4j-log4jSlf4j2Impl = { group = "org.apache.logging.log4j", name = "log4j-slf4j2-impl", version.ref = "apacheLoggingVersion" }
+softwareAmazonAwscdk-awsCdkLib = "software.amazon.awscdk:aws-cdk-lib:2.171.1"
+softwareConstructs-constructs = "software.constructs:constructs:10.4.2"
```

Comment:
These are the new dependencies, but I wanted to sort some of the other dependencies too to get a more consistent ordering.